certified-connectors/VirusTotal/apiProperties.json

```diff
@@ -0,0 +1,21 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "x-api_key",
+          "description": "The x-api_key for this api",
+          "tooltip": "Provide your x-api_key",
+          "constraints": {
+            "tabIndex": 1,
```

Comment:
@ytognder Please change it to 2 as per the latest version of your connector.

"tabIndex": 1 -> "tabIndex": 2

Reply:
![image](https://user-images.githubusercontent.com/60838829/90108174-90865180-dd52-11ea-8482-1f442860a101.png)

@ShyamSunder90  request to change it to 1  see attach image.